### PR TITLE
Added version number to the footer

### PR DESCRIPTION
**Pull request by AyliasTheCoder**
Added the version number to the footer to address issue #228

### CODE REVIEW

**Comment by NayamAmarshe** [comment]
Thanks a lot for the PR! Could you please add a screenshot so I can take a look?

**Comment by AyliasTheCoder** [comment]
image

**Comment by AyliasTheCoder** [comment]
I updates it to be inline with the rest of the text

**Comment by AyliasTheCoder** [comment]
I updated it to be inline with the rest of the footer

image

File: renderer/components/Footer.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import packageJSON from "../../package.json";
```
Comment (NayamAmarshe) [inline renderer/components/Footer.tsx:2]: Interesting that you directly imported the version from `package.json`, it's a nice idea. While it works, I think importing the whole JSON file might be a bit unnecessary.

If you take a look at index.tsx page, we're using the following snippet to fetch the app version: `setVersion(navigator?.userAgent?.match(/Upscayl\/([\d\.]+\d+)/)[1]);`. This makes sure that we're only using what we need and not importing the whole package.json (which might or might not be available during the build step).

File: renderer/components/Footer.tsx [hunk 2]
```diff
@@ -8,23 +9,27 @@ function Footer() {
         <a
           className="font-bold"
           href="https://github.com/upscayl/upscayl"
-          target="_blank">
+          target="_blank"
+        >
           Upscayl
-        </a>
+        </a>{" "}
+        (version <span className="font-bold">{packageJSON.version}</span>)
```
Comment (NayamAmarshe) [inline renderer/components/Footer.tsx:16]:
```suggestion
        (v<span className="font-bold">{navigator?.userAgent?.match(/Upscayl\/([\d\.]+\d+)/)[1]}</span>)
```

**Review by NayamAmarshe — CHANGES_REQUESTED** [review]

File: renderer/components/Footer.tsx (hunk 1 above)
Comment (NayamAmarshe) [inline renderer/components/Footer.tsx:2]:
```suggestion
```

Comment (NayamAmarshe) [reply renderer/components/Footer.tsx:2]: Just one last import to be removed :)

**Comment by NayamAmarshe** [comment]
Looks good! Thanks😁

**Comment by NayamAmarshe** [comment]
Well, I'd have to unfortunately undo these changes because I forgot I already added a version in the header. We don't need another in footer 😅

**Pull request merged**